java-checks/src/test/files/checks/NoTestInTestClassCheck.java

```diff
@@ -87,7 +88,13 @@ public void test2() {
 public class MyNewTest { // should not raise an issue
 }
 @org.junit.runner.RunWith(Enclosed.class)
-public class MyNewTest2 { // no issue
+public class MyNew2Test { // no issue
+}
+@org.junit.runner.RunWith(cucumber.api.junit.Cucumber.class)
```

Comment:
@m-g-sonar Not sure it's relevant but what about testing `@RunWith...` (without full name)?